docs/System_toolchain.rst

```diff
@@ -0,0 +1,148 @@
+.. _system_toolchain:
+
+``system`` toolchain
+====================
+
+In EasyBuild v4.0, the infamous ``dummy`` toolchain was deprecated, and replaced with the ``system`` toolchain.
+
+This page provides more information about this change, and how to deal with the transition.
+
+.. note:: To easy the transition from ``dummy`` to ``system``, EasyBuild v3.9.4 already supports using the ``system``
+          toolchain as an alias for the ``dummy`` toolchain.
+
+.. contents::
+    :depth: 1
+    :backlinks: none
+    :local:
+
+
+.. _system_toolchain_semantics:
+
+Use case for the ``system`` toolchain
+-------------------------------------
+
+The :ref:`system_toolchain` is intended for the same use case as the ``dummy`` toolchain was: it should (only) be used
+for software installations where the compiler & libraries provided by the operating system are assumed to
+be sufficient.
+
+This includes installing binary software (for which no compiler is used at all), installations that boil down to
+only unpacking a source tarball, and building and installing a (base) toolchain compiler like GCC.
+
+For all intents and purposes, the ``system`` **toolchain can be considered as a drop-in replacement for
+the (deprecated)** ``dummy`` **toolchain** (see also :ref:`system_toolchain_impact_deprecated_dummy`).
+
+.. _system_toolchain_usage:
+
+Using the ``system`` compiler
+-----------------------------
+
+To use the ``system`` toolchain in an easyconfig file, simply use ``'system'`` as toolchain name:
+
+.. code:: python
+
+  toolchain = {'name': 'system', 'version': ''}
+
+.. note:: The toolchain version you specify is totally irrelevant when using the ``system`` compiler,
+          it could be ``'system'``, ``''`` or any other (string) value.
+          
+          EasyBuild does not interpret the version of the ``system`` toolchain in any way
+          (as opposed to the ``dummy`` toolchain in EasyBuild versions prior to version 4.0,
+          see :ref:`system_toolchain_motivation_deprecating_dummy_version_and_deps`).
+
+
+.. _system_toolchain_SYSTEM_constant:
+
+``SYSTEM`` constant
+-------------------
+
+For convenience, a ``SYSTEM`` constant is available to easily specify the use of the ``system`` toolchain
+(and to make it stand out against using a regular toolchain):
+
```

Comment:
It should be mentioned that the constant is the preferred way to do this.

Reply:
good point, consider it done